pkgs/development/python-modules/html-table-parser-python3/default.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  poetry-core,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "html-table-parser-python3";
+  version = "0.3.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "schmijos";
+    repo = "html-table-parser-python3";
+    rev = "v${version}";
+    hash = "sha256-okYl0T12wVld7GVbFQH2hgEVKXSScipJN/vYaiRVdGY=";
+  };
+
+  nativeBuildInputs = [ poetry-core ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pytestFlagsArray = [ "test/" ];
```

Comment:
Does it not find the tests or does it find something else, which you are trying to prevent? Add a comment.

Reply:
I assumed that pytest only searched a `tests/` directory by default, not just `test/`. Updated since this flag isn't needed.